packages/node_modules/@ciscospark/internal-plugin-encryption/test/integration/spec/kms.js

```diff
@@ -91,6 +91,89 @@ describe(`Encryption`, function() {
         }));
     });
 
+    /**
+     *Test getAuthorizationList function
+     *Setup: Create a resource, then authorize a user to a key, then authorize a resource to a key
+     *Test:
+     *1)Invoke getAuthorizationList on the resource, verify that the function returns the array that contains two authorized items
+     *2)Remove the authorized user from the resource, then invoke getAuthorizationList again, verify that the returned array doesn't have the user.
+     *3)Remove the authorized resource from the resource again, then invoke getAuthorizationList again, verify that he returned array doesn't have the resource.
+     */
+    describe(`#getAuthorizationList()`, () => {
+      let boundedKeyUri, kro, otherKro;
+      const testUserId = mccoy.spark.internal.device.userId;
+      let testResourceId;
+
+      before(`creates a resource`, () => spark.internal.encryption.kms.createUnboundKeys({count: 1})
+        .then(([key]) => spark.internal.encryption.kms.createResource({
+          key
+        }))
+        .then((k) => {
+          kro = k;
+          boundedKeyUri = kro.keyUris[0];
+          assert.lengthOf(kro.authorizationUris, 1);
+        }));
+
+      before(`authorizes a user to a key`, () => spark.internal.encryption.kms.addAuthorization({
+        userIds: [testUserId],
+        kroUri: kro.uri
+      })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, testUserId);
+          return assert.isFulfilled(mccoy.spark.internal.encryption.kms.fetchKey({uri: boundedKeyUri}));
+        }));
+
+      before(`authorizes a resource to a key`, () => spark.internal.encryption.kms.createUnboundKeys({count: 1})
+        .then(([key]) => spark.internal.encryption.kms.createResource({key}))
+        .then((k) => {
+          otherKro = k;
+          testResourceId = otherKro.uri;
+          return spark.internal.encryption.kms.addAuthorization({
+            authIds: [testResourceId],
+            kro
+          });
+        })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, otherKro.uri);
+          return spark.internal.encryption.kms.getAuthorizationList({kroUri: kro.uri})
+            .then((authIds) => {
+              assert.equal(authIds.length, 2);
+              assert.notEqual(authIds.indexOf(testUserId), -1);
+              assert.notEqual(authIds.indexOf(testResourceId), -1);
+            });
+        }));
+
```

Comment:
remove extra blank line

Reply:
updated the code and removed the extra blank line